<launch>
  <arg name="reportFaultsRate" default="1.0"/>

  <node name="health_monitor_node" pkg="health_monitor" type="health_monitor_node" output="screen">
    <param name="report_faults_rate" type="double" value="$(arg reportFaultsRate)"/>
  </node>

</launch>
